### DISCUSSION TITLE
Sort checkins by actual departure time rather than scheduled time

### DISCUSSION THREAD

**Discussion by marhei** [Traewelling/traewelling#2409] [Ideas / Feature Requests]
Check-ins are currently sorted in the dashboard and on profiles according to the scheduled departure time. This can lead to an incorrect chronological order of the check-ins, especially if connections are not reached due to major delays. This makes it difficult to understand at first glance how the journey was made.

In the screenshot, for example, the last check-in actually took place between the first two due to the delay.
<img width="558" alt="Capture d’écran 2024-03-10 à 19 38 30" src="https://github.com/Traewelling/traewelling/assets/10304183/86ae3aca-c8fb-4c9e-9c06-93ba41e3a980">

**Comment by MrKrisKrisu** [comment]
duplicate of #1382
